middleware.js

```diff
@@ -49,6 +61,24 @@ module.exports.fetchRandomGameData = async (req, res, next) => {
   }
 };
 
+module.exports.fetchRandomGameDataArr = async (req, res, next) => {
+  const pageNumArr = generateUniqueRandomArr();
+  let gameDataArr = [];
+  for (let j = 0; j < 5; j++) {
+    response = await axios.get(
+      `${apiUrl}games?key=${process.env.RAWG_KEY}&page=${pageNumArr[j]}&${apiParams}`
+    );
+    for (let i = 0; i < response.data.results.length; i++) {
```

Comment:
You can use `forEach` method here instead of a `for` loop. Would look more clear and cleaner.

Reply:
Definitely would look tidier that way, will change it